dandiapi/api/tests/test_asset_paths.py

```diff
@@ -295,7 +295,7 @@ def test_asset_path_search_asset_paths(draft_version_factory, asset_factory):
 
 
 @pytest.mark.django_db()
-def test_asset_path_publish_version(draft_version_factory, asset_factory):
+def test_asset_path_publish_version(draft_version_factory, asset_factory, user_factory):
```

Comment:
For the record, if you just need any user you can just use the `user` fixture and avoid the call to `user_factory()` in the test itself.

Reply:
Thanks for the tip--done in 38d1a29a0ef66dd4c987b378d920a9fb8105910c.